pkg/cmd/flags/capture.go

```diff
@@ -18,25 +18,21 @@ Possible options:
 [artifact:]exec                    capture executed files.
 [artifact:]module                  capture loaded kernel modules.
 [artifact:]mem                     capture memory regions that had write+execute (w+x) protection, and then changed to execute (x) only.
-[artifact:]net=interface           capture network traffic of the given interface. Only TCP/UDP/ICMP protocols are currently supported.
 [artifact:]network                 capture network traffic. Only TCP/UDP/ICMP protocols are currently supported.
 
 dir:/path/to/dir                    path where tracee will save produced artifacts. the artifact will be saved into an 'out' subdirectory. (default: /tmp/tracee).
 profile                             creates a runtime profile of program executions and their metadata for forensics use.
 clear-dir                           clear the captured artifacts output dir before starting (default: false).
-pcap:[per-container|per-process]    capture separate pcap file based on container/process context (default: none - saving one pcap for the entire host).
```

Comment:
Default was previously none, meaning no pcap separation was made when this option wasn't chosen. I think we should stick with that.

Reply:
Yes, it is a TODO (number 3) at https://github.com/aquasecurity/tracee/issues/2152. I'll do in a new PR (since this one is about removing old code).

Reply:
added a comment with a TODO (to be soon resolved by me)

Reply:
https://github.com/aquasecurity/tracee/issues/2587